### ISSUE TITLE
CUDA out of memory

### ISSUE THREAD

**Issue by jc-brandeis** [gelles-brandeis/tapqir#359]
I get this error with 113 AOIs, 2 channels, and 1169 frames COSMOS-HMM fitting.  The models converges but will not write the matlab parameter file.   

87%
87000/100000 [44:50:51<6:33:18, 1.82s/it]
/home/jchung/anaconda3/envs/tapqir-env/lib/python3.8/site-packages/torch/distributions/gamma.py:71: UserWarning: Specified kernel cache directory could not be created! This disables kernel caching. Specified directory is /home/jchung/.cache/torch/kernels. This warning will appear only once per process. (Triggered internally at  ../aten/src/ATen/native/cuda/jit_utils.cpp:860.)
  self.rate * value - torch.lgamma(self.concentration))
Iteration #87000 model converged.
Fitting the data: Done
Computing stats ...
- credible intervals
- spot probabilities
ERROR: Failed to compute stats - CUDA out of memory. Try to use smaller AOI/frame batch size

**Comment by ordabayevy** [comment]
Hi Johnson, can you try using smaller AOI batch size? And since fitting is already done you can set the number of iterations to 1 so that it immediately computes stats.

**Comment by gelles-brandeis** [comment]
I think this one is corrected in more recent versions of tapqir.
